src/styles/globals.css

```diff
@@ -193,3 +195,55 @@ html{
 .mouse-cursor-gradient-tracking > * {
   transition: opacity 0.2s ease-in-out;
 }
+
+.gradient-text {
+  background: linear-gradient(
+    72.57deg,
+    #ec008c 17.15%,
+    #5b10ff 89.41%,
+    #0468d6 102.56%
+  );
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-clip: text;
+  text-fill-color: transparent;
+}
```

Comment:
is this still being used ? would it be possible to remove it

Reply:
yess!! Removed